modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -7,6 +7,7 @@
 		List of core built-in GDScript functions. Math functions and other utilities. Everything else is provided by objects. (Keywords: builtin, built in, global functions.)
 	</description>
 	<tutorials>
+		<link title="Random number generation">https://docs.godotengine.org/en/latest/tutorials/math/random_number_generation.html</link>
```

Comment:
I chose to reference the tutorial in @GDScript since it has several methods related to random number generation such as `randi()`.